deliverables/README.md

```diff
@@ -33,36 +35,52 @@ What is the role of Javascript in a web page?
 Why do we separate HTML and CSS?  
 
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+**What are elements?**  
+represents a fragment of computer code.
+**What are tags?**  
+hidden keywords within a web page that define how the browser must format and display the content  
+**What are attributes?**  
+Attributes provide additional information about HTML elements
+**What are forms?**  
+allows a user to enter data that is sent to a server for processing.  
+**What is a div?**  
+Allow the user to divide the page or text
+
+
+**What are selectors?**  
+ are patterns used to select the element(s) you want to style.   
```

Comment:
I like the idea of thinking of them of patterns.  I would advice against using w3schools.com as a reference (http://www.w3schools.com/cssref/css_selectors.asp), I would steer you toward https://developer.mozilla.org which has much better and more up to date info https://developer.mozilla.org/en-US/docs/Glossary/CSS_Selector